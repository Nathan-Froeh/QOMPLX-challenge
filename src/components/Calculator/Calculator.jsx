import React, { Component } from 'react';
import './Calculator.scss';
import Input from '../Input/Input';
import Output from '../Output/Output';

class Calculator extends Component {

  render() {
    return (
      <div className='calculator'>
        <Input/>
        <Output/>
      </div>
    )
  }
}

export default Calculator
