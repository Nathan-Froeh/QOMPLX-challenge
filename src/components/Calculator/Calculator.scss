.calculator {
  height: 400px;
  width: 300px;
  border: #CACACA solid 1px;
  background: #DDDDDD;
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  align-content: center;
  
  form {
    height: 60%;
    width: 90%;
    display: flex;
    flex-wrap: wrap;
    justify-content: flex-end;

    input,
    select {
      height: 30px;
      width: 90%; 
      font-size: 15px;
    }
  
    .submit {
      width: 30%;
    }
  }

  .output {
    height: 30%;
    width: 90%;
    display: flex;
    flex-wrap: wrap;

    div {
      height: 20px;
      width: 100%;
      display: flex;
      justify-content: space-between;

      label,
      p {
        text-align: center;
        margin: 0;
      }
    }
  }
}