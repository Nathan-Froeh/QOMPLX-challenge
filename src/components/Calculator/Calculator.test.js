import React from 'react';
import Calculator from './Calculator';
import {shallow} from 'enzyme';

describe('Calculator', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Calculator/>)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})