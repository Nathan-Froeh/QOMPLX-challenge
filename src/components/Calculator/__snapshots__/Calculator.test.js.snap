// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator should match snapshot 1`] = `ShallowWrapper {}`;
